### DISCUSSION TITLE
VWeb - how to extract get and post-data?

### DISCUSSION THREAD

**Discussion by floscodes** [vlang/v#15569] [General]
Hi folks,

just started trying vlang and vweb and I was wondering if there is a method to get data from the user's request? POST-Params, GET-Params and so on... OR just get the URI-String...

Best regards!

flo

**Comment by floscodes** [comment]
already got it!

**Reply by enghitalo** [reply]
You can see https://github.com/vlang/v/tree/master/vlib/vweb#--parameters
